src/AndroidDebugLauncher/InstallPaths.cs

```diff
@@ -45,38 +45,57 @@ public static InstallPaths Resolve(CancellationToken token, AndroidLaunchOptions
                 ndkRoot = GetDirectoryFromRegistry(RegistryRoot.Value + @"\Setup\VS\SecondaryInstaller\AndroidNDK", "NDK_HOME", checkBothBitnesses: false, externalProductName: LauncherResources.ProductName_NDK);
             }
 
+            NdkReleaseId ndkReleaseId = new NdkReleaseId();
             string ndkReleaseVersionFile = Path.Combine(ndkRoot, "RELEASE.TXT");
-            if (!File.Exists(ndkReleaseVersionFile))
+            string ndkSourcePropertiesFile = Path.Combine(ndkRoot, "source.properties");
+
+            // NDK releases < r11 have a RELEASE.txt file
+            if (File.Exists(ndkReleaseVersionFile))
+            {
+                NdkReleaseId.TryParseFile(ndkReleaseVersionFile, out ndkReleaseId);
+            }
+            // NDK releases >= r11 have a source.properties file
```

Comment:
Might want to check for the new one first in case someone has a crufty install that left the old file around.